DSharpPlus/Exceptions/DiscordException.cs

```diff
@@ -24,27 +24,29 @@
 using System;
 using DSharpPlus.Net;
 
-namespace DSharpPlus.Exceptions
+namespace DSharpPlus.Exceptions;
+
+public abstract class DiscordException : Exception
 {
-    public abstract class DiscordException : Exception
-    {
-        /// <summary>
-        /// Gets the request that caused the exception.
-        /// </summary>
-        public virtual BaseRestRequest WebRequest { get; internal set; }
+    /// <summary>
+    /// Gets the request that caused the exception.
+    /// </summary>
+    public virtual BaseRestRequest WebRequest { get; internal set; } = null!;
+
+    /// <summary>
+    /// Gets the response to the request.
+    /// </summary>
+    public virtual RestResponse WebResponse { get; internal set; } = null!;
 
-        /// <summary>
-        /// Gets the response to the request.
-        /// </summary>
-        public virtual RestResponse WebResponse { get; internal set; }
+    /// <summary>
+    /// Gets the JSON message received.
+    /// </summary>
+    public virtual string? JsonMessage { get; internal set; } = null!;
```

Comment:
nit: `= null!;` is unnecessary here

Reply:
Compiler warning. I could readjust the constructor though.

Reply:
Wait nevermind this is on a nullable property